analysis/wwz/get_wwz_yields.py

```diff
@@ -546,8 +661,8 @@ def make_syst_plots(histo_dict,grouping_mc,grouping_data,save_dir_path,year):
         syst_var_lst = []
         all_syst_var_lst = histo.axes["systematic"]
         for syst_var_name in all_syst_var_lst:
-            if syst_var_name.endswith("Up"):
-                syst_name_base = syst_var_name.replace("Up","")
+            if syst_var_name.endswith("Down"):
+                syst_name_base = syst_var_name.replace("Down","")
```

Comment:
What's the reason for this change? If there's an up there should be a down. Why does it matter to use Down not up?

Reply:
Some of the JEC uncertainties end in 'Up' (PileUp), but none end in Down so far :)